c2corg_ui/templates/helpers.html

```diff
@@ -110,3 +110,43 @@
 <%def name="show_title(title)">\
 ${title + ' - ' if title else ''}Camptocamp.org\
 </%def>
+
+<%def name="show_associated_waypoints(document, type=None)">\
+<ul>
+% if 'waypoints' in document['associations']:
+  % for w in document['associations']['waypoints']:
+    % if type == None:
```

Comment:
What about simply using:
```
% if type is None or type == w['waypoint_type']:
```
=> this will save the other block which has the same content.